src/components/MapView/Layers/ImpactLayer/index.tsx

```diff
@@ -95,7 +95,7 @@ const ImpactLayer = ({ classes, layer }: ComponentProps) => {
 
   return (
     <GeoJSONLayer
-      before={`layer-${boundaryId}-line`}
+      before={before || `layer-${boundaryId}-line`}
```

Comment:
Technically this should not be needed, since `layer-${boundaryId}-line` will be below the labels, so if this layer is below it, it will also be below the layers

Reply:
And doing this might actually shuffle the existing order for other layers

Reply:
Yep, I fix it